docu/examples/example8-uncaught-runtime-errors.ipf

```diff
@@ -6,17 +6,23 @@
 
 Function TestWaveOp()
 
-		WAVE/Z/SDFR=$"I dont exist" wv;
+		WAVE/Z/SDFR=$"I dont exist" wv
+		print "This will be printed, even if a RTE occurs."
+
+		WAVE/Z/SDFR=$"I dont exist" wv; AbortOnRTE
+		print "This will not be printed, as AbortOnRTE aborts the test case."
 End
 
 Function TestWaveOpSelfCatch()
 
 	try
 		WAVE/Z/SDFR=$"I dont exist" wv; AbortOnRTE
+		// If an RTE happens, the execution will jump to catch.
 		PASS()
 	catch
-		// Do not forget to clear the RTE
-		variable err = getRTError(1)
+		print "Here you can print additional info to understand the RTE."
+		// There is no need to clear the RTE (e.g. with GetRTError(1) )
+		// RunTest will take care and print the error message.
 		FAIL()
```

Comment:
Add in the description that the FAIL prevents further execution after it and generates an additional error.